src/Icons/src/Iconify.php

```diff
@@ -66,15 +74,15 @@ public function fetchIcon(string $prefix, string $name): Icon
 
     public function fetchSvg(string $prefix, string $name): string
     {
-        $content = $this->http
-            ->request('GET', sprintf('/%s/%s.svg', $prefix, $name))
-            ->getContent()
-        ;
+        $response = $this->http->request('GET', sprintf('/%s/%s.svg', $prefix, $name));
```

Comment:
I'm not sure a non 200 is ever returned. Is this covered by a test?

Reply:
Yes if the set does not exists / is not available. That's why i added the tests :) 

* https://iconify.design/docs/api/collections.html#error-response
* https://iconify.design/docs/api/collection.html#error-response
* https://iconify.design/docs/api/icon-data.html#error-response

Reply:
Okay, I couldn't remember the exact scenarios were a 404 wasn't returned when I thought it should be.

Reply:
When the prefix exists but not the icon i think (within the collection endpoint)